demo/client/backend/mock/server.go

```diff
@@ -59,8 +59,11 @@ func main() {
 	flag.Parse()
 
 	if flagDebug {
-		// TODO find a way to set flogger log level to debug
-		//logger.SetLevel(shim.LogDebug)
+		// We activate our log level here, note that we can also directly control
```

Comment:
(reading the comments below) if flogger already relies on FABRIC_LOGGING_SPEC, do we still need the flagDebug here?

Reply:
NO - not really needed anymore, however I suggest we keep it as it is and once we decide to revamp the mock server and use it somewhere else for testing we can refactor it and get rid of it.